docs/standard/serialization/binary-serialization.md

```diff
@@ -72,11 +85,44 @@ As the nature of binary serialization allows the modification of private members
 - <xref:System.Collections.Specialized.StringCollection?displayProperty=nameWithType>   
 - <xref:System.Collections.Specialized.StringDictionary?displayProperty=nameWithType>   
 - <xref:System.Collections.Stack?displayProperty=nameWithType>   
+- <xref:System.Colllections.Generic.NonRandomizedStringEqualityComparer?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
```

Comment:
The link doesn't resolve, since we haven't reflected on the type. Can you fence it and add a comment, like this:
\`System.Collections.Generic.NonRandomizedStringEqualityComparer\` \<!--zz \<xref:System.Collections.Generic.NonRandomizedStringEqualityComparer?displayProperty=fullName> -->